apps/dh/app-dh/src/app/datahub-app.module.ts

```diff
@@ -14,15 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NgModule } from '@angular/core';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { DhCoreShellModule } from '@energinet-datahub/dh/core/shell';
+import { NgModule } from '@angular/core';
+
+import { MsalService } from '@energinet-datahub/dh/auth/msal';
 
 import { DataHubAppComponent } from './datahub-app.component';
 
 @NgModule({
   bootstrap: [DataHubAppComponent],
   declarations: [DataHubAppComponent],
   imports: [BrowserAnimationsModule, DhCoreShellModule],
+  providers: [MsalService],
```

Comment:
suggestion: Can we try providing the `MsalService` inside `DhCoreShellModule`? This way we can avoid allowing the "app" tag to depend on a "feature" tag in `.eslintrc.json`.